### PR TITLE
Refactor minor things

### PR DESCRIPTION
**Pull request by fermadeiral**
This PR proposes minor refactorings, mainly related to constants and small util methods.
The build is passing in https://github.com/fermadeiral/repairnator/pull/15

### CODE REVIEW

File: repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java [hunk 1]
```diff
@@ -355,10 +355,10 @@ private void mainProcess() {
 
         ProjectInspector inspector;
 
-        if (config.getLauncherMode() == LauncherMode.BEARS) {
-            inspector = new ProjectInspector4Bears(buildToBeInspected, this.config.getWorkspacePath(), serializers, this.notifiers);
-        } else {
+        if (config.getLauncherMode() == LauncherMode.REPAIR) {
```
Comment (surli) [inline repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java]: Why did you change that if? It seems to me more interesting to check if it's bears or not: the default case should be repair...

Comment (fermadeiral) [reply repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java]: Because in line 344 there is an if for `LauncherMode.REPAIR`. I just wanted to make the two ifs to match to be easier to understand the code.

Comment (fermadeiral) [reply repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java]: I changed the if back, and I put the other if comparing with `LauncherMode.BEARS` then.
